contracts/origination/OriginationController.sol

```diff
@@ -501,13 +392,13 @@ contract OriginationController is
     // solhint-disable-next-line code-complexity
     function _validateLoanTerms(LoanLibrary.LoanTerms memory terms) internal virtual view {
```

Comment:
Could we save gas by having this function be part of `OriginationConfiguration`? It's looks like it's a `view` that only relies on the whitelist state so I think we could. Otherwise we have the gas overhead of all the external contract calls in this function

Reply:
good call, saves 1381 gas per the gas reporter.